feedhenry/fh-fhc
# [fh-fhc] - Unable to run the task fh:dist

## JIRA
https://issues.jboss.org/browse/RHMAP-20288

## WHAT
Unable to run the task fh:dist after new Japanese translations

## WHY
Error with the current version of grunt and its dependences related to compress/parse Japanese encoding. 

## HOW
Upgrade grunt and its dependencies 

## USAGE:
Grunt is used in the dev environment and in CI to execute the tasks as for example `grunt fh:dist`

## STEPS TO TEST
All tasks will be executed when the PR is made. Check if the checks of this PR finished with success. 
Also, you can run locally `grunt dist`.
